extensions/amp-video/0.1/amp-video.js

```diff
@@ -129,11 +129,11 @@ export class AmpVideo extends AMP.BaseElement {
    * @nocollapse
    */
   static prerenderAllowed(element) {
-    // Only allow prerender if video sources are cached on CDN, or if video has
-    // a poster image.
+    // Only allow prerender if video sources are cached on CDN or remote video
+    // cache, or if video has a poster image.
 
-    // Poster is available.
-    if (element.getAttribute('poster')) {
+    // Poster is available, or cache is configured.
+    if (element.getAttribute('poster') || element.hasAttribute('cache')) {
```

Comment:
Should this be checking for `i-amphtml-video-cached-source` instead?

Reply:
This attribute is added later in the code execution (sources are fetched through XHR)